<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Marc Noguera-Julian, PhD. TreeTopUnder" />

<meta name="date" content="2022-04-21" />

<title>Análisis de microbioma fecal en ensayo clínico.</title>

<script src="site_libs/header-attrs-2.13/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Affinity_Mar2022</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Exploratory_v1.html">Exploratorio</a>
</li>
<li>
  <a href="Longitudinal_v1.html">Longitudinal</a>
</li>
<li>
  <a href="Conclusiones_v1.html">Conclusiones</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Análisis de microbioma fecal en ensayo clínico.</h1>
<h4 class="author">Marc Noguera-Julian, PhD. TreeTopUnder</h4>
<h4 class="date">2022-04-21</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-04-21
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>Affinity_Mar2022/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version 1.7.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220409code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20220409)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220409code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220409)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomTreeTopUnderAffinityMar2022tree083253f0e412f1edc6e03108cb87cca45c762b72targetblank083253fa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/TreeTopUnder/Affinity_Mar2022/tree/083253f0e412f1edc6e03108cb87cca45c762b72" target="_blank">083253f</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomTreeTopUnderAffinityMar2022tree083253f0e412f1edc6e03108cb87cca45c762b72targetblank083253fa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/TreeTopUnder/Affinity_Mar2022/tree/083253f0e412f1edc6e03108cb87cca45c762b72" target="_blank">083253f</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/Conclusiones_v1.Rmd
    Ignored:    analysis/Exploratory_v1.Rmd
    Ignored:    analysis/Longitudinal_v1.Rmd
    Ignored:    analysis/Neutered_v1.Rmd
    Ignored:    data/.DS_Store
    Ignored:    data/ps_silva.rds
    Ignored:    data/ps_silva_RR.rds
    Ignored:    data/raw/
    Ignored:    output/.DS_Store
    Ignored:    output/Deliverables/
    Ignored:    output/boxplots/

Untracked files:
    Untracked:  MetadataExplorer.html

Unstaged changes:
    Modified:   .gitignore
    Modified:   analysis/_site.yml
    Modified:   analysis/index.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/index.Rmd</code>) and HTML (<code>docs/index.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/TreeTopUnder/Affinity_Mar2022/blob/083253f0e412f1edc6e03108cb87cca45c762b72/analysis/index.Rmd" target="_blank">083253f</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-09
</td>
<td>
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="datos-del-proyecto" class="section level1">
<h1>Datos del Proyecto</h1>
<p>Propuesta de análisis de datos de microbioma fecal por 16s en ensayo clinico. El ensayo consta de una intervención nutricional en un grupo de perros. En un subgrupo de perros se realiza el cambio de dieta a dieta seca o dieta húmeda de tipo 1 o tipo 2 (variable treatment: d/w1/w2). La clasificación de la dieta que se subministra al animal también se clasifica en una variable binaria (dry_wet) que toma los valores d(dry) o w(wet) Asimismo, cada animal puede estar esterilizado o no (variable neutered Y/N)</p>
<p>Codigo TreeTopUnder: AffinityDogs-16S</p>
<p>Cliente:</p>
<ul>
<li>VetGenomics – Olga Francino Martí</li>
<li>Parc de Recerca de la UAB.Edifici Eureka. Campus UAB. 08193 Bellaterra. Barcelona. Spain.</li>
<li>ContactoÑ:Olga Francino Martí (<a href="mailto:direccio@vetgenomics.com" class="email">direccio@vetgenomics.com</a>)</li>
</ul>
</div>
<div id="objetivo" class="section level1">
<h1>Objetivo</h1>
<p>El objetivo del estudio es caracterizar los cambios en el microbioma intestinal canino en perros que se someten a un cambio de dieta.</p>
</div>
<div id="métodos" class="section level1">
<h1>Métodos</h1>
<div id="analisis-de-datos-de-secuencia." class="section level2">
<h2>Analisis de datos de secuencia.</h2>
<p>Los datos crudos recibidos (formato FASTQ) se han obtenido usando un dieseño de amplicon unico en la region 16S del gen del RNA ribosomico (rDNA), amplificando la sub-region variable V3-V4 mediante PCR y usando secuenciación de última generación (Illumina/MiSeq) sobre las librerias obtenidas. Asi, las lecturas crudas obtenides tienen una longitud de 300 nt en un esquema paired-end (300 nt en lecturas forward y 300 nt en lecturas en reverse) sobre un amplicon de entre 450 y 500 nt, por lo que las lecturas forward y reverse tienen cierto grado de solapamiento que se usa para mejorar la calidad de los datos. En base a los datos crudos obtenidos de muestras orales (saliva) de los pacientes se ha realizado el análisis de secuencia dentro del framework R::DADA2. Este software nos permite obtener secuencias exactas de las variantes estudiadas (Amplicon Sequence Variants o ASV) y una informació de contaje y asignación taxonómica de cada una de ellas. El proceso de análisi se secuencia se divide en los siguientes pasos:</p>
<ul>
<li>Control de calidad. Se eliminan las secuencias con demasiado error acumulado en la lectura forward o reverse, de froma independiente. Se eliminan las secuencias demasiados cortas.</li>
<li>Se calcula el modelo de error especifico del experimento de secuenciación.</li>
<li>Se utilitza el modelo de error obtenido para inferir las variantes ASV a partir de las lecturas</li>
<li>Se solapan las lecturas forward y reverse para obtener una secuencia unica por fragmento.</li>
<li>Se eliminan del dataset las secuencias quimericas provenientes de reacciones PCR incompletas.</li>
<li>Se asigna cada una de las variantes ASV a un grupo taxonomico, usando la base de datos Silva (v138)</li>
<li>Usando las secuencias genéticas de cada una de las variantes ASV se calcula un alineamiento multiple y un arbol filogenético que relaciona evolutivamente las ASV entre ellas.</li>
<li>Se empaquetan los resultados en un paquete de datos tipo R::phyloseq para el análisis posterior.</li>
</ul>
<p>De esta forma, obtenemos un paquete de datos que contiene entidades ASV representativas de los diferentes taxones bacterianos presentes en la muestra y, para cada muestra, valores de contaje cuantitativo de cada uno de estos taxones. Ademas, obtenemos la distancia genética entre pares de taxones que nos permitirá calcular indiced de similitud entre las muestras.</p>
<p>El numero de secuencias obtenidas para este proceso, después de todos los pasos de proceso y control de calidad debe ser suficiente para cada muestra para poder caracterizar el microbioma de la muestra.</p>
<p>Se usan estimadores Chao1 y Shannon para quantificar los paramteros de riqueza y diversidad de la muestra(alpha-diversity). Se usan tests no-parametricos para comparar parametros de alfa-diversidad y beta-diversidad entre grupos formados por las variables de interés. Se usan estimadores Chao1 y Shannon para quantificar los paramteros de riqueza y diversidad de la muestra(alpha-diversity). Se usa distancia WUnifrac para calcular similitud entre la composición de las muestras (Beta-diversity). Se usan non-metric dimensional scaling (NMDS) para el análisis de ordenacion basandose en la distancia WUnifrac. Se usa el paquete lefser para determinar generos bacterianos diferencialmente abundantes entre grupos formados por las variables de interés.</p>
</div>
</div>
<div id="resultados" class="section level1">
<h1>Resultados</h1>
<div id="metadatos" class="section level2">
<h2>Metadatos</h2>
<p><em>Deliverables: FinalMetadata.xls y 16SDataAvailability.xls</em></p>
<p>Cada uno de los ficheros FASTQ esta identificado con un código del formato <em>11754AAC-AFF-2A</em> que identifica una muestra de heces. Existen datos de secuenciación para 53 muestras, incluyendi controles y muestras de heces recogidas durante el ensayo. Se proporcionan metadatos para 47 identificadores muestra de heces esta ligada a un <em>Code</em> que identifica la muestra y un <em>Individuo</em> que identifica al animal del que se han obtenido las heces. Una de las muestras corresponde a un animal (dog_name=“Charlotte”) que solo ha proporcionado una muestra. Esta muestra se eliminara de los análisis apareados. Además, para cada muestra se dispone de las siguientes variables:</p>
<ul>
<li>ID: Identificador numérico del animal</li>
<li>dog_name: Nombre del animal</li>
<li>treatment: Tipo de dieta (d/w1/w2). “d” denota dieta basada en pienso seco. “w1” denota dieta basada en combinación de pienso seco y pienso húmedo. “w2” denota dieta basada en combinación de pienso seco y doble proporción de pienso humedo.</li>
<li>dry_wet: Agregacion de la variable treatment en d/w</li>
<li>T0_bw: Peso corporal basal</li>
<li>size: Tamaño del animal (<em>S</em>mall / <em>M</em>edium)</li>
<li>neutered: Booleano relativo a la castración del animal</li>
<li>BCS:</li>
<li>time_point: punto temporal (t0 / t1). “t0” denota el punto temporal antes de la intervencion (cambio de dieta) y t1, el punto temporal despues del cambio de dieta segun el diseño del estudio.</li>
</ul>
<p>De las 53 muestras disponibles, 6 muestras pertenecen a controles que no se usaran para el análisis mas alla del control de calidad:</p>
<ul>
<li>MOCK, son controles positivos <em>Mock</em> con una composición conocida y predeterminada.</li>
<li>BA, PCR1, PCR2 y Ext corresponden con un blanco ambiental y dos blancos de PCR y uno de extracción.</li>
<li>Neg: Control Negativo.</li>
</ul>
<p>En el dataset resultante, se usaran exclusivamente las ASV anotadas taxonómicamente a Bacteria o Arqueas (Kingdom = Archaea o Kingdom = Bacteria)</p>
<p><img src="figure/index.Rmd/Consolidacio%CC%81n%20de%20datos%20y%20metadatos-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Las caracteristicas del dataset a analizar son:</p>
<ul>
<li>47 muestras de microbioma fecal de perros: ** 24 de punto t0 y 23 de punto t1</li>
<li>1 muestra mock con composición conocida.</li>
<li>un total de 2841 ASV incluyendo singleton, al final del análisi de secuencia</li>
<li>Asignación taxonomica hasta nivel de genero bacteriano para cada ASV</li>
<li>Arbol filogenético asociado a las taxas(ASV), necesario para calcular distancias de beta-diversidad WUnifrac.</li>
</ul>
<pre><code>phyloseq-class experiment-level object
otu_table()   OTU Table:         [ 2841 taxa and 47 samples ]
sample_data() Sample Data:       [ 47 samples by 23 sample variables ]
tax_table()   Taxonomy Table:    [ 2841 taxa by 6 taxonomic ranks ]
phy_tree()    Phylogenetic Tree: [ 2841 tips and 2839 internal nodes ]</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.1 (2021-08-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] forcats_0.5.1    stringr_1.4.0    dplyr_1.0.8      purrr_0.3.4     
 [5] readr_2.1.2      tidyr_1.2.0      tibble_3.1.6     ggplot2_3.3.5   
 [9] tidyverse_1.3.1  phylosmith_1.0.6 here_1.0.1       pander_0.6.5    
[13] phyloseq_1.38.0 

loaded via a namespace (and not attached):
  [1] utf8_1.2.2             tidyselect_1.1.2       grid_4.1.1            
  [4] gmp_0.6-4              Rtsne_0.15             munsell_0.5.0         
  [7] codetools_0.2-18       effectsize_0.6.0.1     units_0.8-0           
 [10] withr_2.5.0            colorspace_2.0-3       Biobase_2.54.0        
 [13] highr_0.9              knitr_1.37             rstudioapi_0.13       
 [16] stats4_4.1.1           ggsignif_0.6.3         rJava_1.0-6           
 [19] labeling_0.4.2         git2r_0.30.1           GenomeInfoDbData_1.2.7
 [22] polyclip_1.10-0        farver_2.1.0           datawizard_0.3.0      
 [25] rhdf5_2.38.0           rprojroot_2.0.2        vctrs_0.3.8           
 [28] generics_0.1.2         xfun_0.30              BWStest_0.2.2         
 [31] R6_2.5.1               GenomeInfoDb_1.30.1    graphlayouts_0.8.0    
 [34] bitops_1.0-7           rhdf5filters_1.6.0     cachem_1.0.6          
 [37] reshape_0.8.8          assertthat_0.2.1       promises_1.2.0.1      
 [40] scales_1.1.1           ggraph_2.0.5           gtable_0.3.0          
 [43] multcompView_0.1-8     xlsx_0.6.5             tidygraph_1.2.0       
 [46] workflowr_1.7.0        rlang_1.0.2            zeallot_0.1.0         
 [49] PMCMRplus_1.9.3        splines_4.1.1          prismatic_1.1.0       
 [52] broom_0.7.12           yaml_2.3.5             reshape2_1.4.4        
 [55] modelr_0.1.8           backports_1.4.1        httpuv_1.6.5          
 [58] RMySQL_0.10.23         tools_4.1.1            logging_0.10-108      
 [61] ellipsis_0.3.2         jquerylib_0.1.4        biomformat_1.22.0     
 [64] RColorBrewer_1.1-2     proxy_0.4-26           BiocGenerics_0.40.0   
 [67] WRS2_1.1-3             Rcpp_1.0.8.3           plyr_1.8.6            
 [70] zlibbioc_1.40.0        classInt_0.4-3         RCurl_1.98-1.6        
 [73] viridis_0.6.2          correlation_0.8.0      S4Vectors_0.32.3      
 [76] haven_2.4.3            ggrepel_0.9.1          cluster_2.1.2         
 [79] fs_1.5.2               magrittr_2.0.2         data.table_1.14.2     
 [82] reprex_2.0.1           mvtnorm_1.1-3          whisker_0.4           
 [85] xlsxjars_0.6.1         hms_1.1.1              patchwork_1.1.1       
 [88] evaluate_0.15          readxl_1.3.1           IRanges_2.28.0        
 [91] gridExtra_2.3          compiler_4.1.1         ggstatsplot_0.9.1     
 [94] KernSmooth_2.23-20     crayon_1.5.0           htmltools_0.5.2       
 [97] mgcv_1.8-39            mc2d_0.1-21            later_1.3.0           
[100] tzdb_0.2.0             RcppParallel_5.1.5     lubridate_1.8.0       
[103] DBI_1.1.2              SuppDists_1.1-9.7      kSamples_1.2-9        
[106] tweenr_1.0.2           dbplyr_2.1.1           MASS_7.3-55           
[109] sf_1.0-7               boot_1.3-28            Matrix_1.4-0          
[112] ade4_1.7-18            permute_0.9-7          cli_3.2.0             
[115] parallel_4.1.1         insight_0.16.0         igraph_1.2.11         
[118] pkgconfig_2.0.3        statsExpressions_1.3.0 xml2_1.3.3            
[121] paletteer_1.4.0        foreach_1.5.2          bslib_0.3.1           
[124] multtest_2.50.0        XVector_0.34.0         rvest_1.0.2           
[127] digest_0.6.29          parameters_0.17.0      vegan_2.5-7           
[130] Biostrings_2.62.0      rmarkdown_2.13         cellranger_1.1.0      
[133] lifecycle_1.0.1        nlme_3.1-155           jsonlite_1.8.0        
[136] modules_0.10.0         Rhdf5lib_1.16.0        viridisLite_0.4.0     
[139] fansi_1.0.2            pillar_1.7.0           lattice_0.20-45       
[142] fastmap_1.1.0          httr_1.4.2             survival_3.3-1        
[145] glue_1.6.2             bayestestR_0.11.5      iterators_1.0.14      
[148] ggforce_0.3.3          class_7.3-20           stringi_1.7.6         
[151] sass_0.4.0             performance_0.8.0      rematch2_2.1.2        
[154] memoise_2.0.1          Rmpfr_0.8-7            e1071_1.7-9           
[157] ape_5.6-2             </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
