<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Marc Noguera-Julian, PhD. TreeTopUnder" />

<meta name="date" content="2022-04-21" />

<title>Análisis longitudinal y Dieta</title>

<script src="site_libs/header-attrs-2.13/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Affinity_Mar2022</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Exploratory_v1.html">Exploratorio</a>
</li>
<li>
  <a href="Longitudinal_v1.html">Longitudinal</a>
</li>
<li>
  <a href="Conclusiones_v1.html">Conclusiones</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Análisis longitudinal y Dieta</h1>
<h4 class="author">Marc Noguera-Julian, PhD. TreeTopUnder</h4>
<h4 class="date">2022-04-21</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-04-21
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>Affinity_Mar2022/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version 1.7.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is ignored by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220409code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20220409)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220409code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220409)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomTreeTopUnderAffinityMar2022tree11b2ba48e433601bdb816d9d741a631326548a12targetblank11b2ba4a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/TreeTopUnder/Affinity_Mar2022/tree/11b2ba48e433601bdb816d9d741a631326548a12" target="_blank">11b2ba4</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomTreeTopUnderAffinityMar2022tree11b2ba48e433601bdb816d9d741a631326548a12targetblank11b2ba4a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/TreeTopUnder/Affinity_Mar2022/tree/11b2ba48e433601bdb816d9d741a631326548a12" target="_blank">11b2ba4</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/Conclusiones_v1.Rmd
    Ignored:    analysis/Exploratory_v1.Rmd
    Ignored:    analysis/Longitudinal_v1.Rmd
    Ignored:    analysis/Neutered_v1.Rmd
    Ignored:    data/.DS_Store
    Ignored:    data/ps_silva.rds
    Ignored:    data/ps_silva_RR.rds
    Ignored:    data/raw/
    Ignored:    output/.DS_Store
    Ignored:    output/Deliverables/
    Ignored:    output/boxplots/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Longitudinal_v1.Rmd</code>) and HTML (<code>docs/Longitudinal_v1.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/TreeTopUnder/Affinity_Mar2022/b5a5ac04d7b96f6fe8c3a0cf546978c65ae2fccf/docs/Longitudinal_v1.html" target="_blank">b5a5ac0</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-21
</td>
<td>
update
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>Como objetivo principal del estudio procedemos a analizar el factor longitudinal de las muestras fecales obtenidas. Cada muestra esta ligada un identificador del animal del que se ha obtenido (dog_name) lo cual nos permite diseño estadísticos apareados. En el conjunto de muestras encontramos muestras apareadas del mismo animal a diferentes tiempos (time_point: t0 o t1), antes y despues de la intervención nutricional. Esta intervencion consta en un cambio de dieta al suministrar diferentes tipos de pienso que, segun la variable treatment son:</p>
<ul>
<li>treatment=d, pienso seco.</li>
<li>treatment=w1, pienso seco + pienso humedo</li>
<li>treatment=w2, pienso seco + (2x)pienso humedo</li>
</ul>
<p>La variable treatment esta agrupada en una variable anidada, dry_wet que toma el valor dry en el caso de perros con <em>treatment=d</em> y el valor “wet” en el caso de perros con <em>treatment=w1</em> o <em>treatment=w2</em>. De la misma forma partimos de la variable neutered(castración) que por el análisis anterior parecen no tener un efecto significativo en la composición del microbioma previa al cambio nutricional.</p>
<div id="alfa-diversidad" class="section level2">
<h2>Alfa-Diversidad</h2>
<p>La alfa diversidad de cada una de las muestras ha sido calculada previamente, en base a los datos disponibles de secuencia. Nos preguntamos:</p>
<ul>
<li>Hay cambios longitudinales en la alfa-diversidad en cada uno de los grupos de tratamiento ( <em>treatment</em> )?</li>
<li>Hay cambios longitudinales en la alfa-diversidad en cada uno de los grupos de tratamiento ( <em>dry_wet</em>)?</li>
</ul>
<p><em>Deliverables: AlphaDiversity.pdf</em></p>
<p>Realizando los análisis en cada uno de los subgrupos:</p>
<ul>
<li>No se detectan cambios significativos en los valores de alfaaa-diversidad (Chao1, Shannon, PD) entre t1 y t0 en el grupo de dieta seca (“d)”</li>
<li>Se detecta un a reducción signifivativa en los valores de diversidad filogenética(PD) entre t1 y t0 en el grupo de dieta húmeda “w1”.</li>
<li>Se detectan un aumento en la alfa-diversidad (Shannon) y una reducción en la diversidad filogenética (PD) entre t1 y t0 en el grupo de dieta húmeda “w2”.</li>
</ul>
<p><img src="figure/Longitudinal_v1.Rmd/alfa-diversity%20vs%20treatment-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-alfa-diversity-vs-treatment-1">
Past versions of "alfa-diversity vs treatment-1.png"
</button>
</p>
<div id="fig-no-spaces-alfa-diversity-vs-treatment-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/Affinity_Mar2022/blob/b5a5ac04d7b96f6fe8c3a0cf546978c65ae2fccf/docs/figure/Longitudinal_v1.Rmd/alfa-diversity vs treatment-1.png" target="_blank">b5a5ac0</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="beta-diversidad" class="section level2">
<h2>Beta-Diversidad</h2>
<div id="evolución-longitudinal" class="section level3">
<h3>Evolución longitudinal</h3>
<p>Usando el mismo método de anàlisis que en la sección previa, usamos la similitud entre la composición del microbioma fecal para calcular los indices de beta-diversidad (WUnifrac, Bray, Unifrac) entre muestras de un mismo perro (paired beta-diversity). Cada perro esta asociado a un grupo de tratamiento o variable de interés, por lo que podemos ver si estos valoroes de paired beta-diversity son diferentes en un grupo u otro.</p>
<p><em>Deliverables: BetaDiversityEvolution_per_group.pdf</em></p>
<p>No se observan cambios en beta-diversidad diferenciales entre grupos de tratamiento (dry_wet, treatment) usando los indices WUnifrac y Bray. Si se observan usando el indice Unifrac (Unweighted). El indice Unifrac(Unweighted) tiene en consideración la similitud filogenetica de las taxas que componene le microbioma pero no su abundancia relativa, lo que si utilizan WUnifrac y Bray.</p>
<p>En el caso de la variable neutered (castración) no se observan cambios usando ningún indice de beta-diversidad.</p>
<p><img src="figure/Longitudinal_v1.Rmd/Paired%20Beta%20Diversity%20per%20variable-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Paired-Beta-Diversity-per-variable-1">
Past versions of "Paired Beta Diversity per variable-1.png"
</button>
</p>
<div id="fig-no-spaces-Paired-Beta-Diversity-per-variable-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/Affinity_Mar2022/blob/b5a5ac04d7b96f6fe8c3a0cf546978c65ae2fccf/docs/figure/Longitudinal_v1.Rmd/Paired Beta Diversity per variable-1.png" target="_blank">b5a5ac0</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dieta-y-cambios-longitudinales" class="section level3">
<h3>Dieta y cambios longitudinales</h3>
<p><em>Deliverables: Adonis_Longitudinal.xls </em></p>
<p>Para testear estadísticamente la influencia del tratamiento/Dieta en los cambios longitudinales de la estructura composicional del microbioma fecal usamos el test de ANOVA permutacional (vegan::adonis) contra la variable time_point en cada uno de los sub-grupos definidos por <em>treatment</em> o <em>dry_wet</em></p>
<p>Tendremos, pues, el resultado para cada uno de los grupos, que nos revelaran cambios asociados al punto temporal, especificos para cada grupo de dieta.</p>
<p><em>Deliverables: Adonis_vs_time_point_per_treatment.xls y Adonis_vs_time_point_per_dry_web.xls</em></p>
<p>Se observa relacion del tipo de dieta con los cambios estructurales en la composición del microbioma a lo largo de la intervencion.</p>
<ul>
<li>En los grupos segregades por <em>treatment</em> la variable time_point (antes y despues de la intervención) explica un 34% de la composicion del microbioma en el grupo con dieta w2, aunque no es significativo en los grupos <em>d</em> y <em>w1</em>.</li>
<li>En la misma tendencia, en los grupos segregadors por <em>dry_web</em> la variable time_point(antes y después de la intervencion) explica un 18% de la composición del microbioma en el grupo <em>w</em>, aunque no es significativa en el grupo <em>d</em>.</li>
</ul>
<p>Estos resultados indican que hay un cambio global en la estructura composicional del microbioma de los perros asociado a la dieta, especificamente, a la dieta <em>w2</em>.</p>
</div>
</div>
<div id="abundancia-diferencial." class="section level2">
<h2>Abundancia diferencial.</h2>
<p>En este análisis se responderá a las preguntas relacionadas con el cambio conretos, a nivel de género bacteriano, de la composición del microbioma en relación a la intervencion nutricional</p>
<div id="variable-treatment" class="section level3">
<h3>Variable <em>treatment</em></h3>
<p>Comparativa no paramétrica entre puntos temporales t1 vs t0, estratificada por grupos de tratamiento (d/w &amp; d/w1/w2)</p>
<p>Comparativa (Wilcoxon) entre la abundancia de diferentes géneros bacterianos entre puntos temporales y la valoración del <em>effect size</em> en la diferencia mediante el paquete <em>lefser</em>.</p>
<p><em>Deliverables: lefser_t1_vs_t0_d_samples.pdf, lefser_t1_vs_t0_w1_samples.pdf, lefser_t1_vs_t0_w2_samples.pdf y boxplots independientes para cada género positivo al test</em></p>
<p>Se observan taxas diferencialmente abundantes entre puntos temporales, especialmente en los grupos de tratamiento w2 y w <em>lefser</em>(<a href="http://www.bioconductor.org/packages/release/bioc/html/lefser.html" class="uri">http://www.bioconductor.org/packages/release/bioc/html/lefser.html</a>):</p>
<ul>
<li>Mayor abundancia en perros despues de haber seguido dieta humeda(w2): <em>Blautia, Dorea, Escherichia Shigella, grupo R.torques i R.gnavus</em></li>
<li>Menor abundancia en perros despues de haber seguido dieta húmeda(w2): <em>Peptoclostridium, Megamonas, Prevotella, Alloprevotella, Sutterella</em></li>
</ul>
<p><img src="figure/Longitudinal_v1.Rmd/DA%20vs%20time_point%20all%20treatment%20groups-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DA-vs-time_point-all-treatment-groups-1">
Past versions of "DA vs time_point all treatment groups-1.png"
</button>
</p>
<div id="fig-no-spaces-DA-vs-time_point-all-treatment-groups-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/Affinity_Mar2022/blob/b5a5ac04d7b96f6fe8c3a0cf546978c65ae2fccf/docs/figure/Longitudinal_v1.Rmd/DA vs time_point all treatment groups-1.png" target="_blank">b5a5ac0</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Como ejemplo de generos diferencialmente abundantes, se muestran las abundancias de dos de los generos que mas diferencias presentan entre t0 y t1 para el grupo <em>w2</em>. En el caso del genero <em>Blautia</em> se detecta un incremento consistente en el numero de reads asignados a ese género bacteriano (p&lt;0.07, en test apareado) para todos los animales.</p>
<p><img src="figure/Longitudinal_v1.Rmd/include%20graphics%20Blautia-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-include-graphics-Blautia-1">
Past versions of "include graphics Blautia-1.png"
</button>
</p>
<div id="fig-no-spaces-include-graphics-Blautia-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/Affinity_Mar2022/blob/b5a5ac04d7b96f6fe8c3a0cf546978c65ae2fccf/docs/figure/Longitudinal_v1.Rmd/include graphics Blautia-1.png" target="_blank">b5a5ac0</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>En el otro sentido, el género <em>Faecalibacterium</em>, reduce su presencia o pasa a ser indetectable en <em>t1</em> en el mismo grupo de animales ( <em>w2</em> )</p>
<p><img src="figure/Longitudinal_v1.Rmd/include%20graphics%20Faecalibacterium-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-include-graphics-Faecalibacterium-1">
Past versions of "include graphics Faecalibacterium-1.png"
</button>
</p>
<div id="fig-no-spaces-include-graphics-Faecalibacterium-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/Affinity_Mar2022/blob/b5a5ac04d7b96f6fe8c3a0cf546978c65ae2fccf/docs/figure/Longitudinal_v1.Rmd/include graphics Faecalibacterium-1.png" target="_blank">b5a5ac0</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="variable-dry_wet" class="section level3">
<h3>Variable dry_wet</h3>
<p>Comparativa (Wilcoxon) + analisis LDA de la abundancia de diferentes generos bacterianos entre los grupos formados por las muestras de los perros con dieta dry(d) y wet(w)</p>
<p><em>Deliverables: lefser_t1_vs_t0_w_samples.pdf y boxplots independientes para cada género positivo al test</em></p>
<p>Se observan taxas diferencialmente abundantes entre muestras del grupo de dieta seca (dry) y el grupo de muestras de dieta humeda mediante <em>lefser</em>(<a href="http://www.bioconductor.org/packages/release/bioc/html/lefser.html" class="uri">http://www.bioconductor.org/packages/release/bioc/html/lefser.html</a>).</p>
<ul>
<li>Mayor abundancia en perros despues de haber seguido dieta humeda(w2): <em>Blautia, Dorea, Escherichia Shigella, grupo R.gnavus</em> y otros</li>
<li>Menor abundancia en perros despues de haber seguido dieta húmeda(w2): <em>Megamonas, Prevotella, Erysipelatoclostrdium, Sutterella</em> y otros.</li>
</ul>
</div>
<div id="ordenación-estratificada-por-grupos-de-treatment" class="section level3">
<h3>Ordenación estratificada por grupos de <em>treatment</em></h3>
<p>Usaremos análisis de ordenación para visualizar los cambios en la estructura del microbioma, independientemente, en cada grupo definido por la variable principal <em>treatment</em>.</p>
<p><em>Deliverables: Ordination_by_treatment_by_time_point.pdf</em></p>
<p>Se puede observar un cambio consistente en todos los animales que forman el grupo <em>w2</em> en el espacio de ordenación, ligado a la proporcion de variabilidad explicada en el análisis PERMANOVA anterior. De la misma manera, se puede observar que los cambios en los grupos de <em>treatment</em> restantes no son consistentes y no producen un <em>clustering</em> en el espacio de ordenación.</p>
<p><img src="figure/Longitudinal_v1.Rmd/ordination%20by%20treatment-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-ordination-by-treatment-1">
Past versions of "ordination by treatment-1.png"
</button>
</p>
<div id="fig-no-spaces-ordination-by-treatment-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/Affinity_Mar2022/blob/b5a5ac04d7b96f6fe8c3a0cf546978c65ae2fccf/docs/figure/Longitudinal_v1.Rmd/ordination by treatment-1.png" target="_blank">b5a5ac0</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.1.1 (2021-08-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] vegan_2.5-7                 lattice_0.20-45            
 [3] permute_0.9-7               phylosmith_1.0.6           
 [5] lefser_1.2.0                SummarizedExperiment_1.22.0
 [7] Biobase_2.54.0              GenomicRanges_1.44.0       
 [9] GenomeInfoDb_1.30.1         IRanges_2.28.0             
[11] S4Vectors_0.32.3            BiocGenerics_0.40.0        
[13] MatrixGenerics_1.4.3        matrixStats_0.61.0         
[15] ggforce_0.3.3               ggplot2_3.3.5              
[17] dplyr_1.0.8                 patchwork_1.1.1            
[19] phyloseq_1.38.0            

loaded via a namespace (and not attached):
  [1] workflowr_1.7.0        plyr_1.8.6             igraph_1.2.11         
  [4] splines_4.1.1          gmp_0.6-4              TH.data_1.1-0         
  [7] kSamples_1.2-9         digest_0.6.29          SuppDists_1.1-9.7     
 [10] foreach_1.5.2          htmltools_0.5.2        viridis_0.6.2         
 [13] lmerTest_3.1-3         fansi_1.0.2            magrittr_2.0.2        
 [16] memoise_2.0.1          paletteer_1.4.0        cluster_2.1.2         
 [19] graphlayouts_0.8.0     Biostrings_2.62.0      RcppParallel_5.1.5    
 [22] sandwich_3.0-1         colorspace_2.0-3       ggrepel_0.9.1         
 [25] xfun_0.30              libcoin_1.0-9          crayon_1.5.0          
 [28] RCurl_1.98-1.6         prismatic_1.1.0        jsonlite_1.8.0        
 [31] lme4_1.1-28            zeallot_0.1.0          zoo_1.8-9             
 [34] survival_3.3-1         iterators_1.0.14       ape_5.6-2             
 [37] glue_1.6.2             polyclip_1.10-0        gtable_0.3.0          
 [40] zlibbioc_1.40.0        XVector_0.34.0         DelayedArray_0.18.0   
 [43] statsExpressions_1.3.0 car_3.0-12             Rhdf5lib_1.16.0       
 [46] Rmpfr_0.8-7            abind_1.4-5            scales_1.1.1          
 [49] mvtnorm_1.1-3          DBI_1.1.2              PMCMRplus_1.9.3       
 [52] Rcpp_1.0.8.3           viridisLite_0.4.0      performance_0.8.0     
 [55] units_0.8-0            proxy_0.4-26           datawizard_0.3.0      
 [58] RColorBrewer_1.1-2     modeltools_0.2-23      ellipsis_0.3.2        
 [61] pkgconfig_2.0.3        reshape_0.8.8          farver_2.1.0          
 [64] multcompView_0.1-8     sass_0.4.0             utf8_1.2.2            
 [67] here_1.0.1             tidyselect_1.1.2       labeling_0.4.2        
 [70] rlang_1.0.2            reshape2_1.4.4         later_1.3.0           
 [73] effectsize_0.6.0.1     munsell_0.5.0          tools_4.1.1           
 [76] cachem_1.0.6           cli_3.2.0              generics_0.1.2        
 [79] ade4_1.7-18            evaluate_0.15          biomformat_1.22.0     
 [82] stringr_1.4.0          fastmap_1.1.0          BWStest_0.2.2         
 [85] yaml_2.3.5             rematch2_2.1.2         knitr_1.37            
 [88] fs_1.5.2               tidygraph_1.2.0        purrr_0.3.4           
 [91] ggraph_2.0.5           coin_1.4-2             WRS2_1.1-3            
 [94] nlme_3.1-155           whisker_0.4            correlation_0.8.0     
 [97] compiler_4.1.1         rstudioapi_0.13        e1071_1.7-9           
[100] ggsignif_0.6.3         tibble_3.1.6           tweenr_1.0.2          
[103] afex_1.1-0             bslib_0.3.1            stringi_1.7.6         
[106] highr_0.9              parameters_0.17.0      forcats_0.5.1         
[109] Matrix_1.4-0           classInt_0.4-3         nloptr_2.0.0          
[112] microbiome_1.17.41     multtest_2.50.0        vctrs_0.3.8           
[115] pillar_1.7.0           lifecycle_1.0.1        rhdf5filters_1.6.0    
[118] mc2d_0.1-21            jquerylib_0.1.4        data.table_1.14.2     
[121] bitops_1.0-7           insight_0.16.0         httpuv_1.6.5          
[124] R6_2.5.1               promises_1.2.0.1       KernSmooth_2.23-20    
[127] gridExtra_2.3          codetools_0.2-18       boot_1.3-28           
[130] MASS_7.3-55            assertthat_0.2.1       rhdf5_2.38.0          
[133] rprojroot_2.0.2        withr_2.5.0            multcomp_1.4-18       
[136] GenomeInfoDbData_1.2.7 mgcv_1.8-39            bayestestR_0.11.5     
[139] parallel_4.1.1         grid_4.1.1             class_7.3-20          
[142] tidyr_1.2.0            minqa_1.2.4            rmarkdown_2.13        
[145] carData_3.0-5          Rtsne_0.15             git2r_0.30.1          
[148] sf_1.0-7               numDeriv_2016.8-1.1    ggstatsplot_0.9.1     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
